src/pages-css/products-page-css/layout-handler.css

```diff
@@ -0,0 +1,129 @@
+@import "../compo-css/navbar.css";
+@import "./page-style.css";
+@import url("https://fonts.googleapis.com/css2?family=Orienta&family=Quicksand&display=swap");
+:root {
+    --color-footer-background: #cecece;
+    --color-red: #fc0101;
+    --color-white: #ffffff;
+    --color-dark: #050505;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+    scroll-behavior: smooth;
+}
+
+
+/* page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 5px;
+    grid-template-columns: 1fr 3fr 2fr;
+    column-gap: 5px;
+    row-gap: 5px;
+    grid-template-areas: 
+    "header header header"
+     "sidebar section-1 section-1 " 
+     "footer footer footer";
+    padding: 5px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+}
+
+
+/* *** */
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    grid-area: header;
+    height: 3.7rem;
+}
+
+/* #section-1 {
+    grid-area: section-1;
+    padding-left: 1.6rem;
+    padding-right: 1.6rem;
+} */
```

Comment:
remove unused code

Reply:
ok

Reply:
this is for further code refactoring and dev purpose.